modules/packages/Makefile

```diff
@@ -17,11 +17,20 @@ packages/install:
 	  rm -rf $(PACKAGES_PATH); \
 	  git clone -c advice.detachedHead=false --depth=1 -b $(PACKAGES_VERSION) https://github.com/cloudposse/packages.git $(PACKAGES_PATH); \
 	  rm -rf $(PACKAGES_PATH)/.git; \
-	fi
+	fi 
 
 ## Install package (e.g. helm, helmfile, kubectl)
 packages/install/%: packages/install
-	@make -C $(PACKAGES_PATH)/install $(subst packages/install/,,$@)
+	$(eval $@_package=$(subst packages/install/,,$@)) 
```

Comment:
Let's not use `eval`; shouldn't be necessary

Reply:
let's do something instead like this:

```
@if ! which $(subst packages/install/,,$@) 2>/dev/null; then \
  make -C $(PACKAGES_PATH)/install; \
 fi
```

Reply:
Basically, if app is not found in `PATH` (which we confirm with `which`), then install

Reply:
The other option is to do:
```
@if [ ! -x $(INSTALL_PATH)/$(subst packages/install/,,$@) ]; then \
....
fi
```